R/animate.R

```diff
@@ -0,0 +1,91 @@
+library(magick)
```

Comment:
No need to add library calls in packages (I think it might even considered bad practice); rather, you just annotate functions with their namespace (as you've done elsewhere). This line can just be deleted

Reply:
Fixed in commit [90aba24](https://github.com/angusmoore/arphit/pull/293/commits/90aba24fc683a6bdddb438669085c5753bbaff33)